packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,93 @@
+
+To enable a more customized user authentication experience, Okta has introduced an addition to the [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid) standard called the Interaction Code grant type. This grant type allows apps to manage user interaction with the authorization server without a browser. This is useful when the client has a particular way that it wants to interact with the user and doesn’t need to share an authenticated session with other applications.
+
+The Interaction Code flow consists of a series of interactions between the user and the authorization server, facilitated by the client. Each interaction is called a remediation step and corresponds to a piece of user data required by the authorization server. The client obtains these remediation steps from the Identity engine component of the Okta Authorization Server and prompts the user for the required data to continue the flow.
+
+For example, a user could start an authentication flow by entering only a username, and this flow would prompt the client to request more information, or remediation, as required. Remediation is the direct communication between the client and the Okta Authorization Server without a browser redirect. An example of a remediation step is the client prompting the user for a password or to add a second factor and then sending that information directly to the Okta Authorization Server.
+
+The Interaction Code grant is intended for developers who want a step-by-step remediation user experience without redirecting to an authorization server. This grant type enables you to include [Identity Engine features](https://help.okta.com/en/oie/okta_help_CSH.htm#csh-features), such as passwordless authentication and progressive profiling. See [Redirect authentication vs. embedded authentication](/docs/concepts/redirect-vs-embedded/) for Identity Engine authentication deployment models and [Identity Engine deployment guides](/docs/guides/oie-intro/) for detailed deployment steps.
+
+## The Interaction Code flow
+
+The Interaction Code flow is similar to the [OAuth 2.0 Authorization Code flow with PKCE](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce). All clients are required to pass along a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. Confidential clients such as web apps must also pass a client secret in their authorization request. The user can start the authorization request with minimal information, relying on the client to facilitate the interactions with the Okta Authorization Server to progressively authenticate the user. The series of interactions, which could include multifactor authentication steps, is secured using the [`interaction_handle`](#interaction-handle). After successfully completing the remedial interactions, the client receives an [`interaction_code`](#interaction-code) that the client can then redeem for tokens at the [`/token`](/docs/reference/api/oidc/#token) endpoint.
+
```

Comment:
Had a header here that said Terms or something along those lines. Do you think we don't need a header for the Interaction code and Interaction handle terms?

Reply:
True. Maybe I can put these terms in a table?

Reply:
a table could work, too

Reply:
Netlify link for reference: https://613bb6151d3a153e05eddd7e--reverent-murdock-829d24.netlify.app

Reply:
Updated Netlify link for reference: https://613fa501b346f00b3a21dff3--reverent-murdock-829d24.netlify.app